### DISCUSSION TITLE
My other pages won’t load

### DISCUSSION THREAD

**Discussion by Bunmi2020** [community/community#112292] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello, I used react to develop my website and deployed it on github pages, but the problem is, the home page works, but one of the other pages (video: https://valormotionpictures.com/video) cannot be found. It loads, shows the page and then suddenly goes off and says “404, file can not be found” after few seconds.

I don’t understand why, I have tried various methods to fix it. Kindly help.

My repository is:  
https://github.com/Bunmi2020/valor-motion-pictures

**Comment by DeandrePetersen** [comment]
It seems like the issue might be related to how your routes are configured in your React application and how GitHub Pages handles client-side routing. When you deploy a React application to GitHub Pages, the routing needs to be handled correctly to ensure that all pages can be accessed. 
Since GitHub Pages serves your site from a subdirectory (https://username.github.io/repository/), you may need to set a base URL for your React application. You can do this by adding a basename prop to your Browser Router or Hash Router component.
<<Browser Router basename={process.env.PUBLIC_URL}>
<HashRouter basename={process.env.PUBLIC_URL}>
If setting the base URL doesn't resolve the issue, consider using Hash Router instead of Browser Router for routing. HashRouter uses the hash portion of the URL (e.g., /#/video) to handle client-side routing, which can sometimes work better with GitHub Pages

**Reply by Bunmi2020** [reply]
Thank you for your response @DeandrePetersen 
I did as you recommended, it works fine on desktop, but on my iPhone, it worse… in addition to the navigation problem, the background video of the other pages pops up and play ls fullscreen on page load before it goes off and says can’t open page.

https://github.com/community/community/assets/74682430/58264910-59d9-4f5a-9bf8-e337fb38a779

There is a specific problem with iPhone, some of my images appears out of place, only iPhones

**Reply by DeandrePetersen** [reply]
Sorry, I am not an expert in programming on IOS, maybe there will be someone to help.
